packages/patternfly-4/react-core/src/components/Popover/Popover.tsx

```diff
@@ -167,6 +169,7 @@ export class Popover extends React.Component<PopoverProps, PopoverState> {
   }
 
   storeTippyInstance = (tip: TippyInstance) => {
+    if(this.props.minWidth) tip.popperChildren.tooltip.style.minWidth = this.props.minWidth;
```

Comment:
Shouldn't we use `c_popover_MinWidth` as the default from react-tokens to set this, similar to the default `maxWidth`?

Reply:
@mcoker Will pf-core apply a default `minWidth`? Note that this component sets a default `maxWidth`.

Reply:
@dlabrecq are you referring to this? https://github.com/patternfly/patternfly-next/blob/7649c9c0a40c2e6e4f6f1e4d1b9cdc6e50aeac66/src/patternfly/components/Popover/popover.scss#L37-L38

Reply:
@mcoker Yes, but we pull it from `@patternfly/react-tokens`. 

The Popover component currently sets a default `maxWdith` using `c_popover_MaxWidth` below. I'm just asking if we should also set a default for `minWidth`?

See https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/components/Popover/Popover.tsx#L126

Reply:
@dlabrecq hmm, do you know why we set that? Doesn't seem like either `minWidth` or `maxWidth` would need to be set by default as they're in the CSS. I would expect that we wouldn't apply either unless it's explicitly defined as a prop, then we just apply whatever the prop value is.

Reply:
I don't know the history behind that. If we don't need it, I'm fine with that. 

We should ensure `maxWidth` is removed in the PR, then.

Reply:
`maxWidth` is passed to tippy and set on the instance, I think it has to have that default value.